tests/kafkatest/services/connect.py

```diff
@@ -206,7 +243,7 @@ def __init__(self, context, kafka, files):
     def node(self):
         return self.nodes[0]
 
-    def start_cmd(self, node, connector_configs):
+    def worker_cmd(self, node, connector_configs):
```

Comment:
the drawback of renaming here is that the name `start_cmd` is used with `start`/`start_node` pretty standardly across the rest of this code base

Reply:
Had to rename because of a weird error: 
```
cmd = self.start_cmd(node, remote_connector_configs)
[INFO:2018-01-15 17:26:38,306]: TypeError: start_cmd() takes exactly 2 arguments (3 given)
```
Let's revisit in the future. Right now, renaming was the only way to make this call work generically with both standalone and distributed classes.

Reply:
ack. would probably be good to verify the minimal diff to rename back after we're confident of the current patch (and might reveal more info), but this isn't all that big a deal if it changes, just makes it a bit inconsistent.